fr/lecons/preserver-logiciels-recherche.md

```diff
@@ -0,0 +1,474 @@
+---
+title: "Préserver et rendre identifiables les logiciels de recherche avec Software Heritage"
+slug: preserver-logiciels-recherche
+layout: lesson
+collection: lessons
+date: 2024-11-15
+authors:
+- Sabrina Granger 
+- Baptiste Mélès 
+- Frédéric Santos
+reviewers:
+- Chiara Mainardi
+- Magaye Sall
+- Martin Amouzou
+editors:
+- Daphné Mathelier
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/616
+difficulty: 1 
+activity: sustaining 
+topics: [data-management]
+abstract: Cette leçon présente des modalités d'archivage adaptées aux spécificités du logiciel et à différents profils d'utilisateurs. Les bonnes pratiques pour citer les logiciels, ainsi que des méthodes pour explorer efficacement les fonds de Software Heritage, seront abordées.
```

Comment:
Just a minor change to make this gender-inclusive. Change "utilisateurs" for "utilisateur·ices" at the end of first sentence, like so: 
"Cette leçon présente des modalités d’archivage adaptées aux spécificités du logiciel et à différents profils d’utilisateur·ices."

Reply:
Merci @marie-flesch!

I have made that change.

Also just worked out how to type the _point médian_ with option + shift + 9 so we are happy ☺️